### PR TITLE
Added window.URL data

### PR DESCRIPTION
**Pull request by gauravmahto**
Adding URL data for api.Window

Completes the [Web/API/Window/URL] task from #2467

### CODE REVIEW

File: api/Window.json [hunk 1]
```diff
@@ -5709,6 +5709,153 @@
...
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/URL",
+          "support": {
+            "chrome": [
+              {
+                "version_added": "41"
+              },
+              {
+                "version_added": "8",
+                "version_removed": "41",
```
Comment (ddbeck) [inline api/Window.json]: Testing in a couple browsers myself, it doesn't look like the prefixes have actually been removed yet

File: api/Window.json [hunk 2]
```diff
@@ -5709,6 +5709,153 @@
...
+                "Implemented under the non-standard name <code>webkitURL</code>."
+              ]
+            },
+            "safari": [
+              {
+                "version_added": "7"
+              },
+              {
+                "version_added": "6",
+                "version_removed": "7",
```
Comment (ddbeck) [inline api/Window.json]: Tested Safari myself and it doesn't look like the prefix has been removed yet

File: api/Window.json [hunk 3]
```diff
@@ -5709,6 +5709,153 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/URL",
+          "support": {
+            "chrome": [
+              {
+                "version_added": "41"
+              },
+              {
+                "version_added": "8",
+                "version_removed": "41",
+                "notes": [
```
[the remark this answers is not in the document]
Comment (gauravmahto) [reply api/Window.json]: This was not a *webkit* prefix.
This was an actual property name.
Previously it was: ```window.webkitURL```
Now: ```window.URL```

File: api/Window.json [hunk 4]
```diff
@@ -5709,6 +5709,153 @@
...
+            },
+            "firefox": [
+              {
+                "version_added": "19"
+              },
+              {
+                "version_added": "4",
+                "version_removed": "19",
+                "notes": [
+                  "From Gecko 2 (Firefox 4) to Gecko 18 included, Gecko returned an object with the non-standard <code>nsIDOMMozURLProperty</code> internal type. In practice, this didn't make any difference."
```
[the remark this answers is not in the document]
Comment (gauravmahto) [reply api/Window.json]: OK. Didn't know that.

File: api/Window.json [hunk 5]
```diff
@@ -5709,6 +5709,153 @@
...
+            ],
+            "firefox_android": [
+              {
+                "version_added": "19"
+              },
+              {
+                "version_added": "14",
+                "version_removed": "19",
+                "notes": [
+                  "From Gecko 2 (Firefox 4) to Gecko 18 included, Gecko returned an object with the non-standard <code>nsIDOMMozURLProperty</code> internal type. In practice, this didn't make any difference."
```
[the remark this answers is not in the document]
Comment (gauravmahto) [reply api/Window.json]: OK

File: api/Window.json [hunk 6]
```diff
@@ -5709,6 +5709,153 @@
...
+                ]
+              }
+            ],
+            "ie": {
+              "version_added": "10"
+            },
+            "opera": {
+              "version_added": "15",
+              "notes": [
```
[the remark this answers is not in the document]
Comment (gauravmahto) [reply api/Window.json]: Same as I've mentioned before.

File: api/Window.json [hunk 7]
```diff
@@ -5709,6 +5709,153 @@
...
+                "Implemented under the non-standard name <code>webkitURL</code>."
+              ]
+            },
+            "opera_android": {
+              "version_added": "15",
+              "notes": [
```
[the remark this answers is not in the document]
Comment (gauravmahto) [reply api/Window.json]: Same as I've mentioned before.

File: api/Window.json [hunk 8]
```diff
@@ -5709,6 +5709,153 @@
...
+                "version_added": "6",
+                "version_removed": "7",
+                "notes": [
+                  "Implemented under the non-standard name <code>webkitURL</code>."
+                ]
+              }
+            ],
+            "safari_ios": {
+              "version_added": "6",
+              "notes": [
```
[the remark this answers is not in the document]
Comment (gauravmahto) [reply api/Window.json]: Same as I've mentioned before.

File: api/Window.json (hunk 2 above)
Comment (gauravmahto) [reply api/Window.json]: Ok. This I just used based on the info in the old table. I'll revert it.

File: api/Window.json (hunk 1 above)
Comment (gauravmahto) [reply api/Window.json]: I've had tested on *console* window using this [link](https://turbo.net/run/chrome)
On v40 and the output was: *no deprication notice* 
On v41 it was: *depricated*

**Comment by gauravmahto** [comment]
@ddbeck I've committed the updated code based on your suggestion. Please review. Thanks :)

File: api/Window.json (hunk 1 above)
Comment (ddbeck) [reply api/Window.json]: I still think this line should be deleted. I tested it on Chrome 68 and got the deprecation warning, but `webkitURL` still works. Generally, we use `"version_removed"` to mean "this feature no longer works in this browser" even if the vendor discourages its use. So it hasn't been removed yet, at least for the purposes of our schema

File: api/Window.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (ddbeck) [reply api/Window.json]: I don't think I understand the distinction you're making here. If the standard name of the property is `URL` and the vendor has added `webkit` to the beginning of the property name (making it `webkitURL`), isn't that a vendor prefix?

File: api/Window.json (hunk 1 above)
Comment (gauravmahto) [reply api/Window.json]: OK. I was under different assumptions. Got it now.

File: api/Window.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (gauravmahto) [reply api/Window.json]: Got it. Thanks for the clarification.

**Comment by gauravmahto** [comment]
@ddbeck I've updated the source with new commits. Please review. Thanks!

File: api/Window.json [hunk 9]
```diff
@@ -5709,6 +5709,131 @@
           }
         }
       },
+      "URL": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/URL",
+          "support": {
+            "chrome": {
+              "version_added": "41",
```
Comment (ddbeck) [inline api/Window.json]: I think this got collapsed into one entry erroneously. I think the final result should be:
```
"chrome": [
  {
    "version_added": "41"
  },
  {
    "version_added": "8",
    "prefix": "webkit"
  }
]
```

File: api/Window.json [hunk 10]
```diff
@@ -5709,6 +5709,131 @@
...
+            "samsunginternet_android": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": false,
```
Comment (ddbeck) [inline api/Window.json]: Sorry, I should've caught this before. This is standards track

File: api/Window.json [hunk 11]
```diff
@@ -5709,6 +5709,131 @@
...
+              "samsunginternet_android": {
+                "version_added": null
+              },
+              "webview_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
```
Comment (ddbeck) [inline api/Window.json]: Same here, I'm sorry, I should've caught this before. This is standards track

**Comment by gauravmahto** [comment]
@ddbeck I've updated the source. Please review. Thanks!

File: api/Window.json (hunk 11 above)
Comment (gauravmahto) [reply api/Window.json]: I guess this is correct.

**Comment by connorshea** [comment]
There are a bunch of changes in this PR that shouldn't be included.

**Comment by Elchi3** [comment]
I'm sorry but this is not going to get merged. The page "Web/API/Window/URL" was incorrectly added to MDN and has now been fixed to redirect to https://developer.mozilla.org/en-US/docs/Web/API/URL which already has compat data. Sorry about the inconveniences and thanks for contributing :heart:

**Pull request closed without merging**